backend/.env.bak

```
@@ -0,0 +1,321 @@
+# PAYPAL CONFIGURATION (PRODUCTION SETTINGS)
+# =============================================================================
+# PayPal Environment (sandbox or live)
+PAYPAL_MODE=live
+
+# PayPal API Credentials (REPLACE WITH YOUR ACTUAL CREDENTIALS)
+PAYPAL_CLIENT_ID=AQLxJ2PTOCLuWEzPWRKF3GP-DMl1jbmLMsjk-xu8FqtqJsaXjO36uVX9wOefiewqWzC7_jXkbDyDw2SK
+PAYPAL_CLIENT_SECRET=EBz7lzVK6akwzzRLbmNEPdWbUVA19ihnW194xHOR--dE7EJY7NQo1kMo7JMztwrwNpMGd8CA9LrH68UJ
+
+# PayPal URLs
+PAYPAL_WEBHOOK_URL=https://api.retailtradescanner.com/wp-json/stock-scanner/v1/paypal-webhook
+PAYPAL_RETURN_URL=https://retailtradescanner.com/membership-success/
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The PAYPAL_RETURN_URL key should go before the PAYPAL_WEBHOOK_URL key

(UnorderedKey)